packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.tsx

```diff
@@ -433,4 +439,20 @@ export class SpinButton extends BaseComponent<ISpinButtonProps, ISpinButtonState
     this._updateValue(true /* shouldSpin */, this._initialStepDelay, this._onDecrement!);
   }
 
+  private _getUpArrowButtonStyles() {
+    const { styles: customStyles, theme } = this.props;
+    const customStylesArrowButton = customStyles && customStyles.arrowButtonStyles;
+    const customStylesUpArrowButton = customStyles && customStyles.upArrowButtonStyles;
+
+    return getArrowButtonStyles(theme!, true, customStylesArrowButton, customStylesUpArrowButton);
+  }
+
+  private _getDownArrowButtonStyles() {
+    const { styles: customStyles, theme } = this.props;
+    const customStylesArrowButton = customStyles && customStyles.arrowButtonStyles;
+    const customStylesDownArrowButton = customStyles && customStyles.downArrowButtonStyles;
+
+    return getArrowButtonStyles(theme!, false, customStylesArrowButton, customStylesDownArrowButton);
+  }
```

Comment:
These functions are almost exactly the same consider passing a parameter to determine up vs down styles and collapse to one function